api_tests/package.json

```diff
@@ -4,6 +4,7 @@
     "description": "COMIT node end-to-end testing suite",
     "main": "index.js",
     "scripts": {
+        "build": "./node_modules/typescript/bin/tsc",
```

Comment:
:wrench: 

`tsc` should also be in `node_modules/.bin/tsc`. If that is the case, you can just refer to `tsc` here :)